pom.xml

```diff
@@ -22,7 +22,7 @@
         <maven.compiler.version>3.11.0</maven.compiler.version>
         <maven.deploy.version>2.8.2</maven.deploy.version>
         <maven.release.version>2.5.3</maven.release.version>
-        <okapi.version>1.45.0</okapi.version>
+        <okapi.version>1.46.0</okapi.version>
```

Comment:
TheOkapi framework was updated because it doesn't support SnakeYAML 2.2 (which is updated by Spring Boot 3.2.7) 
https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-3.2-Release-Notes